doc/source/whatsnew/v0.19.2.rst

```diff
@@ -7,6 +7,7 @@ Version 0.19.2 (December 24, 2016)
 
 .. ipython:: python
    :suppress:
+   :okwarning:
```

Comment:
So a star import will raise the `FutureWarning` from `pd.value_counts`? That's kinda unfortunate as it seems noisy

Reply:
i dont see an alternative that isnt behavior-changing

Reply:
i guess could warn inside value_counts if it is called

Reply:
Yeah I think it's more directly useful to warn users who are using `pd.value_counts` directly`

Reply:
updated + greenish